docs/articles/developer-info/Packaging-the-V2-Adapter.md

```diff
@@ -67,13 +67,13 @@ The resulting files can be found in the "package" folder:
 You need Visual Studio 2013.  We use the ultimate edition, but it should be enough with the premium edition.  (I will probably work with both the Pro or the Express editions too, but we haven't tried them).  The latest 1.1 version is built using Update 2 RC.
 
 1. **Visual Studio 2013 SDK**  
-You need this to work with the vsix.  Download from <http://www.microsoft.com/en-us/download/details.aspx?id=40758>
+You need this to work with the vsix.  Download from <https://www.microsoft.com/en-us/download/details.aspx?id=40758>
 
 1. **NAnt**
-Download from <http://nant.sourceforge.net/>.  We use the 0.92 version.
+Download from <https://nant.sourceforge.net/>.  We use the 0.92 version.
 
 1. **NuGet**
-You need the nuget.exe in your path.  Download the exe from <http://nuget.codeplex.com/downloads/get/784779>.  We use the 2.8 version
+You need the nuget.exe in your path.  Download the exe from <https://nuget.codeplex.com/downloads/get/784779>.  We use the 2.8 version
```

Comment:
@Arhell are you personally checking these links to ensure they have `https` editions? This link here, for example, doesn't appear to work anymore at all, so I'm not sure it makes sense to change to https.

Reply:
All check & update